### PR TITLE
1154 new proposal card

### PR DESCRIPTION
**Pull request by brodeur**
https://daostack.tpondemand.com/RestUI/Board.aspx#page=board/5209716961861964288&appConfig=eyJhY2lkIjoiQjgzMTMzNDczNzlCMUI5QUE0RUE1NUVEOUQyQzdFNkIifQ==&boardPopup=feature/1154/silent

### CODE REVIEW

**Comment by jellegerbrandy** [comment]
@brodeur can you fix the test? thanks :-)

File: src/components/Proposal/Predictions/PredictionButtons.tsx [hunk 1]
```diff
@@ -227,46 +230,95 @@ class PredictionBox extends React.Component<IProps, IState> {
...
+              </span>
+            }
+          </div>
         </div>
-      </div>
-    );
+      );
+    } else {
+      return (
+        <div className={wrapperClass}>
```
Comment (tibetsprague) [inline src/components/Proposal/Predictions/PredictionButtons.tsx]: this looks very duplicated from above, do we really need to duplicate all this code? what actually changes for the "context menu"?

File: src/components/Proposal/ProposalCard.scss [hunk 2]
```diff
@@ -43,7 +43,7 @@
   display: inline-block;
   position: relative;
   box-shadow: 0 0 15px 0 rgba(133,168,208,.15);
-  overflow: hidden;
+  // overflow: hidden;
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalCard.scss]: remove if we dont want

File: src/components/Proposal/Voting/VoteButtons.tsx [hunk 3]
```diff
@@ -120,60 +124,139 @@ class VoteButtons extends React.Component<IContainerProps, IState> {
...
+              </span>
+              : " "
+            }
+          </div>
         </div>
+      );
+    } else {
+      return (
+        <div className={wrapperClass} >
+          {this.state.showPreVoteModal ?
```
Comment (tibetsprague) [inline src/components/Proposal/Voting/VoteButtons.tsx]: again id like to figure out what doesnt need to be duplicated, like this probably doesnt

File: src/components/Proposal/Predictions/PredictionButtons.tsx (hunk 1 above)
Comment (brodeur) [reply src/components/Proposal/Predictions/PredictionButtons.tsx]: The proposal card for mobile still uses the other way of voting.. Not sure a more elegant way to do it. Perhaps a mobile design for the new cards will allow us to do away with the duplication? @A-Zak

File: src/components/Proposal/Voting/VoteButtons.tsx (hunk 3 above)
Comment (brodeur) [reply src/components/Proposal/Voting/VoteButtons.tsx]: Happy to look at it together... perhaps they should be two separate components?

**Pull request merged**